src/app/modules/qsa/components/documentation/documentation-info/documentation-info.component.html

```diff
@@ -1,9 +1,13 @@
-<div class="card shadow-sm ">
-	<div class="card-header">
-		<h4 class="my-0 font-weight-normal">{{ 'documentation' | translate}}</h4>
-	</div>
-	<div class="card-body">
-		<div class="card-text">Mire is jó?</div>
-		<div class="card-text">Amennyiben szeretnél külön kéréseket küldeni a szervernek és nyers adatokkal dolgozni, akkor a lent található linkeket használd. Az alkalmazás JSON választ fog adni minden esetben, hibakód viszont a 200-as lesz.</div>
-	</div>
+<div class="card shadow-sm">
```

Comment:
This one is not used at all. I think we can safely remove it.

Reply:
Do you mean the whole component?